apps/ledger-live-mobile/src/index.tsx

```diff
@@ -484,19 +479,14 @@ const getOnboardingLinkingOptions = (acceptedTermsOfUse: boolean) => ({
   },
 });
 
-const platformManifestFilterParams = {
-  private: true,
-  branches: undefined, // will override & having it to undefined makes all branches valid
-};
-
 const DeepLinkingNavigator = ({ children }: { children: React.ReactNode }) => {
   const dispatch = useDispatch();
   const hasCompletedOnboarding = useSelector(hasCompletedOnboardingSelector);
   const wcContext = useContext(_wcContext);
-  const { state: remoteLiveAppState } = useRemoteLiveAppContext();
-  const liveAppProviderInitialized =
-    !!remoteLiveAppState.value || !!remoteLiveAppState.error;
-  const filteredManifests = useFilteredManifests(platformManifestFilterParams);
+  const emptyObject: AppManifest[] = [];
```

Comment:
It's important to move this constant outside the component function to keep the same ref all the time